Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubelet/dockertools/manager.go

```diff
@@ -544,11 +544,12 @@ func (dm *DockerManager) runContainer(
 	}
 
 	hc := &docker.HostConfig{
-		Binds:       binds,
-		NetworkMode: netMode,
-		IpcMode:     ipcMode,
-		UTSMode:     utsMode,
-		PidMode:     pidMode,
+		Binds:          binds,
+		NetworkMode:    netMode,
+		IpcMode:        ipcMode,
+		UTSMode:        utsMode,
+		PidMode:        pidMode,
+		ReadonlyRootfs: readOnlyRootFilesystem(container),
```

Comment:
Yep, I need to rebase this PR to get rid of that commit, it was just so I could code it before the rebase landed.  Ultimately this line goes away with 22857.